gradle-module-plugin/src/functionalTest/kotlin/io/ia/sdk/gradle/modl/BaseTest.kt

```diff
@@ -18,24 +11,29 @@ import java.nio.file.Files
 import java.nio.file.Path
 
 data class SigningResources(
-    val keystore: Path,
-    val certFile: Path,
+    val keystore: Path?,
+    val certFile: Path?,
+    val pkcs11Cfg: Path?,
     val signPropFile: Path?
 )
 
 open class BaseTest {
     companion object {
-        val CLIENT_DEP = "// add client scoped dependencies here"
-        val DESIGNER_DEP = "// add designer scoped dependencies here"
-        val GW_DEP = "// add gateway scoped dependencies here"
-        val COMMON_DEP = "// add common scoped dependencies here"
-        val SIGN_PROPS = "signing.properties"
+        const val CLIENT_DEP = "// add client scoped dependencies here"
+        const val DESIGNER_DEP = "// add designer scoped dependencies here"
+        const val GW_DEP = "// add gateway scoped dependencies here"
+        const val COMMON_DEP = "// add common scoped dependencies here"
         val SIGNING_PROPERTY_ENTRIES = """
-            ${Constants.SIGNING_PROPERTIES[ALIAS_FLAG]}=selfsigned
-            ${Constants.SIGNING_PROPERTIES[KEYSTORE_FILE_FLAG]}=./keystore.jks
-            ${Constants.SIGNING_PROPERTIES[KEYSTORE_PW_FLAG]}=password
-            ${Constants.SIGNING_PROPERTIES[CERT_FILE_FLAG]}=./certificate.pem
-            ${Constants.SIGNING_PROPERTIES[CERT_PW_FLAG]}=password
+            ignition.signing.certAlias=selfsigned
+            ignition.signing.keystorePassword=password
+            ignition.signing.certFile=./certificate.pem
+            ignition.signing.certPassword=password
```

Comment:
No DRY/re-use on the constant values because 1) we want to test those constants too, not just the logic, and 2) clarity test inputs for the reader--More BDD-oriented.